roles/unicorn_user/templates/defaults.j2

```diff
@@ -26,3 +26,5 @@ OFN_DB_USERNAME={{ db_user }}
 {% if db_password is defined %}
 OFN_DB_PASSWORD={{ db_password }}
 {% endif %}
+
+MEMCACHED_VALUE_MAX_MEGABYTES={{ memcached_value_max_megabytes }}
```

Comment:
(sorry, I left this comment some days ago but I didn't submit it :see_no_evil: )

Reply:
> sorry, I left this comment some days ago but I didn't submit it

Probably for a different change/PR, because this was only done a couple of hours ago? :slightly_smiling_face: 

> Isn't this then a duplicate? Or you're trying things out?

The lowercase variable name is because of our Ansible vars convention of using lowercase characters, while the uppercase one is loaded into env and used by the Rails app for which we usually use uppercase. Are you okay with this?

Reply:
More details here: https://github.com/openfoodfoundation/openfoodnetwork/pull/4356#issue-325632663